_includes/footer.html

```diff
@@ -47,24 +47,24 @@
 			<h4 class="text-right"><a href="/contact">Contact us</a></h4>
 			<div class="flex justify-space-between" style="gap: 3px;">
 				<a href="https://github.com/godotengine" target="_blank" rel="noopener">
-					<img src="/assets/footer/github_logo.svg" width="32" height="32" alt="GitHub">
+					<img src="/assets/footer/github_logo.svg" width="32" height="32" alt="GitHub" loading="lazy">
 				</a>
 				<a href="https://mastodon.gamedev.place/@godotengine" target="_blank" rel="noopener">
-					<img src="/assets/footer/mastodon_logo.svg" width="32" height="32" alt="Mastodon">
+					<img src="/assets/footer/mastodon_logo.svg" width="32" height="32" alt="Mastodon" loading="lazy">
 				</a>
 				<a href="https://twitter.com/godotengine" target="_blank" rel="noopener">
-					<img src="/assets/footer/twitter_logo.svg" width="32" height="32" alt="Twitter">
+					<img src="/assets/footer/twitter_logo.svg" width="32" height="32" alt="Twitter" loading="lazy">
 				</a>
 				<a href="https://www.facebook.com/groups/godotengine/" target="_blank" rel="noopener">
-					<img src="/assets/footer/facebook_logo.svg" width="32" height="32" alt="Facebook">
+					<img src="/assets/footer/facebook_logo.svg" width="32" height="32" alt="Facebook" loading="lazy">
 				</a>
 				<a href="https://www.reddit.com/r/godot" target="_blank" rel="noopener">
 					<!-- Zero-width space in the `alt` text to prevent content blockers from blocking the icon -->
-					<img src="/assets/footer/reddit_logo.svg" width="32" height="32" alt="Red​dit">
+					<img src="/assets/footer/reddit_logo.svg" width="32" height="32" alt="Reddit" loading="lazy">
 				</a>
 				<a href="/rss.xml" target="_blank" rel="noopener">
 					<!-- Icon is called `feed` instead of `rss` to prevent content blockers from blocking the icon -->
-					<img src="/assets/footer/feed_logo.svg" width="32" height="32" alt="RSS feed">
+					<img src="/assets/footer/feed_logo.svg" width="32" height="32" alt="RSS feed" loading="lazy">
 				</a>
 			</div>
 		</div>
```

Comment:
I wouldn't use lazy-loading for SVGs as these files are tiny, yet seeing them with a visible delay feels odd.

Reply:
Yes makes sense, I've removed it & also I didn't notice the comment about the zero-width space, fixed as well.